app/lib/github_linkback.rb

```diff
@@ -109,13 +122,23 @@ def self.field_for(url)
 
   private
 
-  def post_pr(link)
-    github_url = "https://api.github.com/repos/#{link.project}/issues/#{link.pr_number}/comments"
-    comment = I18n.t(
-      'github_linkback.pr_template',
-      title: SiteSetting.title,
-      post_url: "#{Discourse.base_url}#{@post.url}"
-    )
+  def post_pr_or_issue(link, type)
+    pr_or_issue_number = link.pr_number || link.issue_number
+    github_url = "https://api.github.com/repos/#{link.project}/issues/#{pr_or_issue_number}/comments"
+    case type
+    when :pr
+      comment = I18n.t(
+        'github_linkback.pr_template',
+        title: SiteSetting.title,
+        post_url: "#{Discourse.base_url}#{@post.url}"
+      )
+    when :issue
+      comment = I18n.t(
+        'github_linkback.issue_template',
+        title: SiteSetting.title,
+        post_url: "#{Discourse.base_url}#{@post.url}"
+      )
+    end
```

Comment:
I wrote the above section this way because calls to translation functions are often statically analyzed. I wasn't sure if this was the case in Discourse or not.

Reply:
They are not, so please do DRY up the method a bit.

Reply:
Done in f43a1d7